### PR TITLE
Movie DB API Javascript

### PR DESCRIPTION
**Pull request by GrahamGolf**
This is the functionality for the Movie DB API call where we will display the results of different searches.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,12 +54,13 @@
     </div>
   </section>
   
+<<<<<<< HEAD
```
Comment (katjones23) [inline index.html]: In VS Code, this is a conflict that needs to be resolved.

File: index.js [hunk 2]
```diff
@@ -1,15 +1,309 @@
...
+     multiSearch(mainSearch);
+        })
  
+    
   
-  document.addEventListener('DOMContentLoaded', function() {
-    var elems = document.querySelectorAll('input-field');
-    var instances = M.FormSelect.init(elems, options);
-  });
+<<<<<<< HEAD
```
Comment (katjones23) [inline index.js]: in VS code this is a conflict that needs to be resolved.

**Review by katjones23 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (GrahamGolf) [reply index.html]: yeah i'm not sure where that came from

File: index.js (hunk 2 above)
Comment (GrahamGolf) [reply index.js]: Also not sure where this is from either. I think it was from when Isaiah started the JS file.

**Pull request closed without merging**